native~/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
This is needed because of the use of `FILE(CONFIGURE)` below. CMake 3.18 came out July 2020, so hopefully this isn't a problem.